ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.2.yaml

```diff
@@ -3,7 +3,7 @@ tag_specification:
   namespace: ocp
 promotion:
   namespace: openshift
-  name: openshift-serverless-v1.3.0
+  name: openshift-serverless-nightly
```

Comment:
I suppose we don't care about these promoted images. Because otherwise it would be good to differentiate 4.1, 4.2, 4.3.

Reply:
I suppose we'd need to change a few other configs as well if we wanted to differentiate. I'd defer that to a later PR in the spirit of getting this in to unblock development.